7_react/react1/week3/vite-project/src/components/addNewToDo.jsx

```diff
@@ -0,0 +1,168 @@
+import { useState, useEffect } from "react";
+import ListItemWithBorder from "./listItemWithBorder.jsx";
+
+const API_URL="https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+
+function AddNewToDo() {
```

Comment:
The name of this component is perhaps a bit misleading as its functionality goes quite beyond just adding new todos. 😄 It is quite a mouthful to take in! Can you think of a way to simplify the component by splitting it up into multiple components? 🙂

Reply:
I will change it to ToDoManaging or I will make separate components :)